### PR TITLE
View all questions

### PR DESCRIPTION
**Pull request by Hammad15**
merging team-2 and team-3 work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "draft-js": "^0.11.7",
     "draftjs-to-html": "^0.9.1",
     "html-react-parser": "^0.13.0",
+    "labeled-checkbox-material-ui": "^2.6.3",
```
Comment (MrNorthmore) [inline package.json:19]: please remove :)

File: package.json [hunk 2]
```diff
@@ -24,7 +25,8 @@
     "react-scripts": "3.4.1",
     "redux": "^4.0.5",
     "redux-thunk": "^2.3.0",
-    "typescript": "~3.7.2"
+    "typescript": "~3.7.2",
+    "use-force-update": "^1.0.7"
```
Comment (MrNorthmore) [inline package.json:29]: plz remove :)

File: src/components/pages/quest-components/quest-container.component.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                 }
+            }else{
+                retrievedPageable = await questionRemote.getAllQuestions(userId, page);
+                if (retrievedPageable.numberOfElements === 0) {
+                    return;
+                }
+            }
+        } else {
+            if(location){
+                //SHOULD BE CHANGED!!!!
```
Comment (HenryHsieh999) [inline src/components/pages/quest-components/quest-container.component.tsx:111]: Already changed. Should delete comment later.

**Review by Colin-Shaw2 — APPROVED** [review]
LGTM!

**Pull request merged**